public_budget/models/budget_modification_detail.py

```diff
@@ -22,13 +22,40 @@ class BudgetModificationDetail(models.Model):
         'public_budget.budget_position',
         string='Budget Position',
         required=True,
+        store=True,
```

Comment:
I wanna know why do you need to add a store True in budget_position_id field denifition?